doc/user/content/cli.md

```diff
@@ -116,6 +117,30 @@ also use this to change the port that Materialize listens on from the default
 The `materialized` [Docker image](/install/#docker) instead uses a listen
 address of `0.0.0.0:6875` by default, in accordance with Docker conventions.
 
+### Cross-origin resource sharing (CORS)
+
+{{< version-added v0.26.0 />}}
```

Comment:
Not important at this stage, but I'm assuming we just never added a release note for this back then (other than a bug fix for it in `v0.26.1`).

Reply:
Ah damn, I should have read the whole thing first!